workers_bindings.go

```diff
@@ -309,6 +311,34 @@ func (b WorkerAnalyticsEngineBinding) serialize(bindingName string) (workerBindi
 	}, nil, nil
 }
 
+// WorkerQueueBinding is a binding to a Workers Queue.
+//
+// https://developers.cloudflare.com/workers/platform/bindings/#queue-bindings
+type WorkerQueueBinding struct {
```

Comment:
How are we differentiating between a `producer` and `consumer` binding? https://developers.cloudflare.com/queues/configuration/

Should we add an optional sub-resource to allow folks to configure the consumer? Something like https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloudfunctions_function#event_trigger

Reply:
This PR is only for the `producer` side of Queues config. I'll tackle all the consumer stuff separately, but yeah we are going to need to define a new type of sub-resource for consumers. Basically I was thinking that a script could have two sub resources: `queue_bindings` and `queue_consumers` (names could be changed...) that would handle each type.

Reply:
@jbw1991 are you planning on tackling queue consumers anytime soon?